src/audio/drc/drc.c

```diff
@@ -34,7 +35,9 @@
 #include <stddef.h>
 #include <stdint.h>
 
-static const struct comp_driver comp_drc;
+#define DISABLED_CODE 0
```

Comment:
this is weird? Why not just remove that code is not used?

Reply:
Yep, it will go away, forgot to remove.